### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by csayaan**
Updating the description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
 # dashboard
 
-Packit-service web UI dashboard.
+# About
```
Comment (TomasTomecek) [inline README.md]: we can drop this line - there already is a main header in this section: `# dashboard`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+ Repository containing the [dashboard](https://github.com/packit-service/dashboard) of [Packit](https://github.com/packit-service) application.
+
+ # Requirements
```
Comment (TomasTomecek) [inline README.md]:
```suggestion
 ## Requirements
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+ [dashboard](https://github.com/packit-service/dashboard) is written in python3 and is support only on 3.6 or later. Always try to use the latest version.
+ ``` bash
```
Comment (TomasTomecek) [inline README.md]: we should suggest that this section contains steps for Ubuntu, so please prepend `### Ubuntu`

File: README.md [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+ ```
+ # Installation
+ At first go to the [dashboard](https://github.com/packit-service/dashboard) repository and clone it into your local machine with the https key.
+ ``` bash
+    $ git clone https://github.com/csayaan/dashboard.git
+ ```
+ Go to the top directory
+ ``` bash
+ cd dashboard/
+ ```
```
Comment (TomasTomecek) [inline README.md]: I'd say these are implied - cloning the repository and being in the root, I'd suggest dropping these.

Comment (csayaan) [reply README.md]: Ok, I will try to fix it.

**Comment by TomasTomecek** [comment]
We're using git pre-commit hooks to improve quality of the files in the repo, you can find instructions how to set it up locally and use it in here: https://github.com/packit-service/packit-service/blob/master/CONTRIBUTING.md#checkerslintersformatters--pre-commit

**Pull request closed without merging**